core/src/main/java/de/danoeh/antennapod/core/service/GpodnetSyncService.java

```diff
@@ -218,11 +256,11 @@ private synchronized void processEpisodeActions(List<GpodnetEpisodeAction> local
                     GpodnetEpisodeAction localMostRecent = localMostRecentPlayAction.get(key);
                     if(localMostRecent == null ||
                             localMostRecent.getTimestamp().before(action.getTimestamp())) {
-                        GpodnetEpisodeAction mostRecent = remoteMostRecentPlayAction.get(key);
+                        GpodnetEpisodeAction mostRecent = mostRecentPlayAction.get(key);
                         if (mostRecent == null) {
-                            remoteMostRecentPlayAction.put(key, action);
```

Comment:
why make these two different branches? they do the same thing?  just an || would be fine